cgnsutilities/cgnsutilities.py

```diff
@@ -2883,9 +2905,31 @@ def simpleCart(xMin, xMax, dh, hExtra, nExtra, sym, mgcycle, outFile):
 # -----------------------------------------------------------------
 # These functions perform operations that return new 'Grid' objects
 # -----------------------------------------------------------------
+def _getBocoDataSetArray(inFile, iBlock, iBoco, iBocoDataSet, BCDataType, iDir):
```

Comment:
Only commenting here, but this applies elsewhere as well. Couple of things,
1. Even though its small, I dont think we should make code changes like this (variable renaming and moving code) in a PR meant for linting (unless its relevant to the linting). Makes blaming less effective, etc. Can you please separate this (and related changes) out into a separate PR?
2. To be consistent with current naming scheme (also too close to class naming), change `BCDataType` to `bcDataType` (and elsewhere).

Reply:
Good point, moved this change to #79

Reply:
Just to check, do you plan to revert this relocation of this function and the changes as well?

Reply:
No, the relocation was necessary to meet one of the ruff linting rules

Reply:
ok sounds good, do you remember what rule this was ?